web/docs/setup/tune.md

```diff
@@ -54,129 +52,69 @@ The `vast.import.read-timeout` option determines how long a call to read data
-VAST considers the partition as complete, regardless of the number of records.
-
-A LRU cache of partitions accelerates queries to recently used partitions. The
-parameter `vast.max-resident-partitions` controls the number of partitions in
-the LRU cache.
+VAST maintains a LRU cache of partitions to accelerate queries involving recent
+partitions. The parameter `vast.max-resident-partitions` controls the number of
+partitions in the LRU cache.
 
 :::note
 Run `vast flush` to force VAST to write all active partitions to disk
 immediately. The command returns only after all active partitions were flushed
 to disk.
 :::
 
-### Catalog
-
-The catalog manages partition meta data and is responsible for deciding whether
-a partition qualifies for a certain query. It does so by maintaining *sketch*
-data structures (e.g., Bloom filters, summary statistics) for each partition.
-Sketches are highly space-efficient at the cost of being probabilistic and
-yielding false positives.
-
-Due to this characteristic sketches can grow sub-linear, doubling the number of
-events in a sketch does not lead to a doubling of the memory requirement.
-Because the catalog must be traversed in full for a given query it needs to be
-maintained in active memory to provide high responsiveness.
+### Tune number of sketches and their sizes
 
-As a consequence, the overall amount of data in a VAST instance and the
-`max-partition-size` determine the memory requirements of the catalog. The
-option `max-partition-size` is inversely linked to the number of sketches in the
+The catalog's memory usage is a function of the stored data. The configuration
+option `max-partition-size` determines an upper bound of the number of records
+per partition, which is inversely linked to the number of sketches in the
 catalog. That means increasing the `max-partition-size` is an effective method
 to reduce the memory requirements for the catalog.
 
 ### Tune sketch parameters
 
-A false positive can have substantial impact on the query latency by
-materializing irrelevant partitions, which involves unnecessary I/O. Based on
-the cost of I/O, this penalty may be substantial. Conversely, reducing the false
-positive rate increases the memory consumption, leading to a higher resident set
-size and larger RAM requirements.
+A false positive during a sparse index lookup can have a noticeable impact on the
+query latency by materializing irrelevant partitions. Based on the cost of I/O,
+this penalty may be substantial. Conversely, reducing the false positive rate
+increases the memory consumption, leading to a higher resident set size and
+larger RAM requirements.
 
 You can control this space-time trade-off in the configuration section
-`vast.index` by specifying index *rules*. Each rule corresponds to one sketch
-and consists of the following components:
+`vast.index` by specifying index *rules*. Each rule configures one sketch and
```

Comment:
Each entry in that list configures potentially multiple catalog or partition indexes, as a rule can refer to multiple targets. Essentially, it allows for:
- Enabling field-specific catalog indexes per extractor (off by default)
- Changing the false-positive rate of catalog indexes per extractor or type for catalog indexes that support a false-positive rate
- Disabling partition indexes for a given extractor

I think there are quite a few holes in that design, but let's document what we got as that is not the current focus.

Reply:
I rewrote the indexing section to reflect this. Please resolve if this works.